docs/_snippets/storybook-debug-webpack-prod.md

```diff
@@ -1,8 +1,8 @@
 ```shell renderer="common" language="js" packageManager="npm"
-npm run build-storybook -- --debug-webpack
+npm run storybook build -- --debug-webpack
```

Comment:
@ahayes91 can you revert this here as well? As it was technically correct as this refers to the script